apis/builder/validators.yaml

```
@@ -0,0 +1,37 @@
+post:
+  operationId: "registerValidator"
+  summary: Register or update a validator's block building preferences.
+  description: |
+    Registers a validator's preferred fee recipient and gas limit.
+
+    Builders should verify that `pubkey` corresponds to an active or pending
+    validator, and that `signature` is valid under `pubkey`. Otherwise, builder
+    must error. Requests with `timestamp` less than or equal to the previous
+    announcement must error. Requests with `timestamp` more than 10 seconds in
```

making sure builders don't track timestamps of "error" calls (both bad sigs and too future of timestamps)
```suggestion
    successful announcement must error. Requests with `timestamp` more than 10 seconds in
```